modules/core/src/main/scala/ch/epfl/scala/debugadapter/internal/StackTraceProvider.scala

```diff
@@ -13,16 +13,14 @@ import java.util.Optional
 
 class StepFilterProvider(
     stepFilters: Seq[StepFilter],
+    scalaStepFilter : ScalaStepFilter,
     logger: Logger,
     testMode: Boolean
 ) extends JavaStepFilterProvider() {
 
-  override def shouldSkipFrame(method: Method): Boolean = {
-    stepFilters.exists(_.shouldSkipOver(method))
 
-  }
   override def formatMethodName(method: Method): Optional[String] = {
```

Comment:
It formats the full signature not only the name.

```suggestion
  override def formatMethod(method: Method): Optional[String] = {
```

Or

```suggestion
  override def formatMethodSig(method: Method): Optional[String] = {
```